src/Tgstation.Server.Host/Utils/SwaggerConfiguration.cs

```diff
@@ -493,7 +493,6 @@ public void Apply(OpenApiDocument swaggerDoc, DocumentFilterContext context)
 			var productHeaderSchema = new OpenApiSchema
 			{
 				Type = "string",
-				Format = "productheader",
```

Comment:
Why was this removed?

Reply:
since it looks odd on the documentation (even though it is already a header)
![image](https://github.com/user-attachments/assets/dcb67cfe-28eb-4162-a1b8-6a97df59a132)